<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1225</width>
    <height>866</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="PlotWidget" name="pressure_plot" native="true">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>250</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="pressure_verticalLayout">
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QLabel" name="pressure_label">
            <property name="text">
             <string>Pressure</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="pressure_show">
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>30</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="temperature_verticalLayout">
            <item>
             <widget class="QLabel" name="temperature_label">
              <property name="text">
               <string>Temperature</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="temperature_show">
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>24</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="ref_pressure_verticalLayout">
            <item>
             <widget class="QLabel" name="ref_pressure_label">
              <property name="text">
               <string>Ref pressure</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="ref_pressure_show">
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>24</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="ref_temperature_verticalLayout">
            <item>
             <widget class="QLabel" name="ref_temperature_label">
              <property name="text">
               <string>Ref Temperature</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="ref_temperature_show">
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>24</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="delta_p_verticalLayout">
            <item>
             <widget class="QLabel" name="delta_p_label">
              <property name="text">
               <string>DeltaP</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="delta_p_show">
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>24</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="recording_time_verticalLayout">
            <item>
             <widget class="QLabel" name="recording_time_label">
              <property name="text">
               <string>Recording time:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="recording_time_show">
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>24</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="start_recording_pushButton">
            <property name="text">
             <string>Start measurement</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stop_recording_pushButton">
            <property name="text">
             <string>Stop measurement</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="comment_label">
          <property name="text">
           <string>Comment</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="comment_lineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1225</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
